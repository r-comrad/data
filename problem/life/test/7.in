asd000123a()(0(()sd4566d5f4g+65d4f6g351s+6df4gdf65g6dfg656df6f5g65d4r65643d1f3g1d65frg1+dfg-d*f/g+41d+fg7-d*f/g+dfg+dfg789fg
