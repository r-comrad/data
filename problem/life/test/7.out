6 5 4 1 3 0 7 9 8 2 
